eureka-client/src/test/java/com/netflix/discovery/util/EurekaEntityFunctionsTests.java

```diff
@@ -0,0 +1,363 @@
+package com.netflix.discovery.util;
```

Comment:
Noticed the file name suffix is plural "Tests", rename to `eureka-client/src/test/java/com/netflix/discovery/util/EurekaEntityFunctionsTest.java` (singular Test) to match the other tests.

Reply:
Done